### DISCUSSION TITLE
MonoTargetsTasksAssemblyPath value is missing

### DISCUSSION THREAD

**Discussion by Take-A-Byte** [dotnet/runtime#100484] [Q&A]
When I am building a library with native AOT for iossimulator-arm64 using the following command
`dotnet build -c Release -r iossimulator-arm64 /p:UseNativeAOTRuntime=true /p:RunAOTCompilation=false /p:NativeLib=Shared`

I get following error - 
```
/usr/local/share/dotnet/packs/Microsoft.NET.Runtime.LibraryBuilder.Sdk/9.0.0-preview.2.24128.5/Sdk/LibraryBuilder.targets(3,14):
error MSB4022: The result "" of evaluating the value "$(MonoTargetsTasksAssemblyPath)" of the "AssemblyFile" attribute in element <UsingTask> is not valid.
```

Does anyone know how to tackle this?

**Comment by elringus** [comment]
I had a similar issue when targeting `browser-wasm`. Running `dotnet workload repair` fixed it for me.
